<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LeakDialogQtUi</class>
 <widget class="QDialog" name="LeakDialogQtUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resource Leak Testing</string>
  </property>
  <widget class="QLabel" name="Testslabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>78</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TESTS:~~~~</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="IterationsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>117</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ITERATIONS:~~~~</string>
   </property>
  </widget>
  <widget class="QLabel" name="BytesPerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BYTES PER:~~~~</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="newOperatorCheckBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>147</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>new operator~~~~</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewOperIterationsLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="placeholderText">
    <string extracomment="Enter number of iterations to perform"/>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewOperBytesPerLineEdit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="RunTestsButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>104</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Run Tests~~~~</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>83</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close~~~~</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
